### ISSUE TITLE
Consider LowDB for database

### ISSUE THREAD

**Issue by zuperm4n** [jesusgn90/bar-commander#11]
**Is your feature request related to a problem? Please describe.**
Initial set up and installation of the system uses MongoDB. Some users may be unfamiliar with the configuration up for this which may limit the number of people willing to use the system - especially whilst it's very new and unknown.

**Describe the solution you'd like**
Consider [https://github.com/typicode/lowdb](https://github.com/typicode/lowdb) which is a NoSQL database based on `.json` files and requires zero config when cloning the repo.

Once the project has made progress and is ready to be used in production, it can be switched back to Mongo.

**Describe alternatives you've considered**
N/A

**Additional context**
Just a suggestion. This project is still very small and needs a lot of development before it is even close to being useful. If we make it as easy as possible for others to contribute the project is more likely to gain traction.

**Comment by zuperm4n** [comment]
Alternatively, create a continuous integration deploy for a development server API on heroku so contributors who don't want to set up a Mongo environment locally.

**Comment by jesusgn90** [comment]
Yes, you are right. What do you think about use it depending on NODE_ENV value? if development let's use LowDB,  otherwise use MongoDB

**Comment by zuperm4n** [comment]
@jesusgn90  `NODE_ENV` is probably the best and most straight-forward way to do this and switch between the two. Not sure how we can sync the data in both database environments though.

**Comment by TolleyB-J** [comment]
or they could use mlab.com free mongo hosting
I use them for some of my projects
